lib/managers/tasks.js

```diff
@@ -85,6 +85,39 @@ Tasks.prototype.get = function(taskID, qs, callback) {
 		};
 
 	this.client.get(apiPath, params, this.client.defaultResponseHandler(callback));
+
+};
+
+/**
+ * Updates a specific task.
+ *
+ * API Endpoint: '/tasks/:taskID'
+ * Method: PUT
+ *
+ * @param {string} taskID - The Box ID of the task being updated
+ * @param {?Object} options - Additional parameters
+ * @param {string} [options.message] - An optional message to include with the task
+ * @param {string} [options.dueAt] - The day at which this task is due
+ * @param {Function} callback - Passed the updated task information if it was acquired successfully, error otherwise
+ * @returns {void}
+ */
+Tasks.prototype.update = function(taskID, options, callback) {
+	var apiPath = urlPath(BASE_PATH, taskID),
+		params = {
+			body: {
+				action: REVIEW_ACTION
+			}
+		};
+
+	if (options && options.message) {
+		params.body.message = options.message;
+	}
+
+	if (options && options.dueAt) {
+		params.body.due_at = options.dueAt;
+	}
```

Comment:
These will need to be updated to the standard options passing style, where all options are added to the body (not just whitelisted ones), e.g. by `Object.assign()`